host/4/bookworm/python/python312/install_ca_certificates.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Source and destination directories
+source_dir="/var/ssl/root"
+destination_dir="/usr/local/share/ca-certificates"
```

Comment:
This is missing AGC stuff. Could you please fix this : 
https://github.com/Azure/azure-functions-docker/blob/dev/host/4/bookworm/dotnet/install_ca_certificates.sh

Could you also please take a look where you copied this from to see if it needs an update I dont think it should.

Reply:
Added this from node. Its still missing there.,

Reply:
Could you please fix both of them

Reply:
Lets not mix node changes in this PR. Can open another PR for node

Reply:
@gavin-aguiar which Node folder is missing AGC certs script?  I will send a separate PR

Reply:
This file does not have the AGC steps. https://github.com/Azure/azure-functions-docker/blob/dev/host/4/bookworm/node/node22/install_ca_certificates.sh